cgi/org.pl

```diff
@@ -218,29 +227,34 @@
 			}
 		}
 
-		push (@$admin_fields_ref, (
-			{
-				field => "list_of_gs1_gln",
-			},
-			{
-				field => "gs1_product_name_is_abbreviated",
-				type => "checkbox",
-			},
-			{
-				field => "gs1_nutrients_are_unprepared",
-				type => "checkbox",
-			},	
-		));
-
-		push @{$template_data_ref->{sections}}, {
+		push(
+			@$admin_fields_ref,
+			(
+				{
+					field => "list_of_gs1_gln",
+				},
+				{
+					field => "gs1_product_name_is_abbreviated",
+					type => "checkbox",
+				},
+				{
+					field => "gs1_nutrients_are_unprepared",
+					type => "checkbox",
+				},
+			)
+		);
+
+		push @{$template_data_ref->{sections}},
+		  {
```

Comment:
I wonder why there are 2 spaces before the { (and not a tab)

Reply:
I added another rule to try to fix the tabs, how does this look?